frontend/test/searchform.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import SearchForm from '../app/components/searchform';
+import { AsyncTypeahead } from 'react-bootstrap-typeahead';
+
+describe('<SearchForm />', () => {
+  it('should render the typeahead search form', () => {
+/*
+    const props = {
+      options: [
+        { name: 'restaurant', camis: '123456', address: 'location'},
+        { name: 'restaurant2', camis: '223456', address: 'location2'}
+      ],
+      renderBusinessOptions: sinon.spy(),
+      handleSearch: sinon.spy()
+    }
+*/
+    const wrapper = shallow(<SearchForm />)
+    expect(wrapper.find(AsyncTypeahead)).to.have.length(1);
+  });
+
+  it('should have props for handleSearch and renderBusinessOptions', () => {
+    const wrapper = shallow(<SearchForm />)
+    expect(wrapper.props().handleSearch).to.be.defined;
+    expect(wrapper.props().renderBusinessOptions).to.be.defined;
+  })
+/*
+  it('should call handleSearch if the user inputs a query', () => {
+    const handleSearchSpy = sinon.spy()
+    const wrapper = mount(<SearchForm handleSearch={handleSearchSpy} />)
+    //simulate user input
+    wrapper.find(AsyncTypeahead).simulate('change', {target: {value: 'my query'}})
+    expect(handleSearchSpy.called).to.equal(true)
+  })
+*/
```

Comment:
this doesn't work. it is me thinking out loud - i'm trying to figure out how to test `handleSearch` will be called on a change when the user types something (not testing the actual functionality of what happens, just that is should be called hence the mock) This makes sense in simple examples where you can simulate a on change or on input in an input/form component that you wrote yourself instead of my case with a complicated typeahead child component. i think i'm just going to get rid of this because its basically trying to test behavior of the typeahead and not the component itself?